### ISSUE TITLE
Remove high runtime unit-tests 

### ISSUE THREAD

**Issue by polybassa** [secdev/scapy#2445]
A evaluation of campaign run-times showed that the following Campaigns cause the highest test duration: 
```
### Loading: test\pipetool.uts
[00:02:54] Campaign CRC=5A33F65B in 021.01s SHA=4B698E79427709CB214266FE8EFA2DFE100A8727
### Loading: test\contrib\automotive\gm\gmlanutils.uts
[00:03:47] Campaign CRC=94F77E87 in 023.22s SHA=88A8D1D4BF14D76A78A7532FA6DA8438645E200E
### Loading: test\regression.uts
[00:03:21] Campaign CRC=EDD0870C in 027.30s SHA=ED367C85074D49EC7B0CA99CA50342B668EC02F6
### Loading: test\contrib\isotp.uts
[00:10:45] Campaign CRC=1C455158 in 061.05s SHA=A3F0B13F191850B4FB8C9CA1E3B2F48C90EA633E
### Loading: test\contrib\isotpscan.uts
[00:10:45] Campaign CRC=55188426 in 315.24s SHA=0819C1757535966A8FC3E333C4D76B1168549184
```
I will try to remove the run-times of automotive related unit tests.

**Comment by polybassa** [comment]
#2456 tries to speed up `test\contrib\isotpscan.uts`

**Comment by polybassa** [comment]
FYI: This command filters the 5 tests with the longest run-time

```wget -q -O - <url of log> | grep -E 'passed' | sort -k 3,3 | tail -n 5```

**Comment by guedou** [comment]
Fixed in #2456
